### ISSUE TITLE
How can i improve accuracy?

### ISSUE THREAD

**Issue by philosophocat** [cazala/synaptic#286]
I'm trying to teach network for [cubic parabola](https://jsfiddle.net/4r8z6f9f/2/). It works, but not very good. How can i improve it? Why it works worse, if the configuration is (2, 6, 6, 2) instead of (2, 6, 2), it should be better, shouldn't it?

**Comment by Pummelchen** [comment]
Play with:

- learning rate
- iterations limit
- target error limit (MSE)
- training data size 
- very deep layers (bit slow)

In my network with 27 inputs and 1 output I have very fast results with 
27 1 1 1 using only 1 training set (row of data) but repeating it quite a lot for very low MSE (> 0.0000001)

Just play with the parameters, after some hours you should get a feeling which settings work best :)
